www/src/pages/foundations/icons.mdx

```diff
@@ -1,8 +1,3 @@
----
-title: Icons
```

Comment:
Agreed, these should be removed but we are losing the page title (`<title>`).

Production: 
Page Title: `Icons | Paragon Design System`

PR Deploy Preview:
Page Title:  no title set

---

Should we migrate these files to a .jsx file and include the following, similar to other non-component pages?

```jsx
{/* eslint-disable-next-line react/jsx-pascal-case */}
<SEO title="Colors" />
```